mlx/coverity_services.py

```diff
@@ -236,14 +246,16 @@ def assemble_query_filter(self, column_name, filter_values, matcher_type):
             "matchers": matchers
         }
 
-    def get_defects(self, stream, filters, column_names):
-        """Gets a list of defects for given stream, filters and column names.
+    def get_defects(self, stream, snapshot, filters, column_names):
```

Comment:
Add new arguments to the end, just in case someone is already using this function...

Reply:
@Letme Do you mean a keyword argument instead of a position argument? 72a6cb16eff49d16ab98c6d658ddd291e8a8431e does not address your concern. Should we be concerned with backwards compatibility in this MR? The plugin was heavily reworked in #74 and will be released with a new major version nr.

Reply:
For me the commit resolves the comment. There is no backwards compatibility of this function at the moment, and I doubt positional argument would fix it, but when migrating, it is easier to spot if new arguments are on last spot and their order is unchanged, than if we now mix (for no real added value) the arguments. Function will still fail in external call because of not enough arguments, but it will be easier to see that snapshot argument needs to be added, than if it is hidden in the middle of other arguments.